second-gen/packages/swc/components/status-light/stories/status-light.stories.ts

```diff
@@ -15,7 +15,7 @@ import '@swc/components/status-light';
 
 export default {
     component: 'swc-status-light',
-    title: 'Components/StatusLight',
+    title: 'StatusLight',
```

Comment:
Interesting- this title has a space between the two words "Progress circle," but no space here for "Status light." Is there a preference? 

I haven't pulled this down yet to check, but if this is the display name in the sidenav, I assume we would want "StatusLight" to be "Status light."